pycgm/pycgm.py

```diff
@@ -0,0 +1,5263 @@
+        axis_results[axis_idx["R HANY"]] = r_hany
+        axis_results[axis_idx["R HANZ"]] = r_hanz
+        axis_results[axis_idx["L HANO"]] = l_hano
+        axis_results[axis_idx["L HANX"]] = l_hanx
+        axis_results[axis_idx["L HANY"]] = l_hany
+        axis_results[axis_idx["L HANZ"]] = l_hanz
+
+        # Populate RHand and LHand in joint_centers
+        joint_centers[jc_idx["RHand"]] = r_hano
+        joint_centers[jc_idx["LHand"]] = l_hano
+
+        # Angle calculations
+        # Pelvis
+        pelvis_angle = pel_an(measurements['GCS'], pelvis_axis)
+        angle_results[angle_idx["Pelvis"]] = pelvis_angle
+
+        # Hip
+        right_hip_angle, left_hip_angle = hip_an(hip_axis, knee_axis)
+        angle_results[angle_idx["R Hip"]] = right_hip_angle
+        angle_results[angle_idx["L Hip"]] = left_hip_angle
+
+        # Knee
+        right_knee_angle, left_knee_angle = kne_an(knee_axis, ankle_axis)
+        angle_results[angle_idx["R Knee"]] = right_knee_angle
+        angle_results[angle_idx["L Knee"]] = left_knee_angle
+
+        # Ankle
+        right_ankle_angle, left_ankle_angle = ank_an(ankle_axis, foot_axis)
+        angle_results[angle_idx["R Ankle"]] = right_ankle_angle
+        angle_results[angle_idx["L Ankle"]] = left_ankle_angle
+
+        # Foot
+        right_foot_angle, left_foot_angle = foo_an(measurements['GCS'], foot_axis)
+        angle_results[angle_idx["R Foot"]] = right_foot_angle
+        angle_results[angle_idx["L Foot"]] = left_foot_angle
+
+        # Head
+        head_angle = hea_an(measurements['GCS'], head_axis)
+        angle_results[angle_idx["Head"]] = head_angle
+
+        # Thorax
+        thorax_angle = tho_an(thorax_axis)
+        angle_results[angle_idx["Thorax"]] = thorax_angle
+
+        # Neck
+        neck_angle = nec_an(head_axis, thorax_axis)
+        angle_results[angle_idx["Neck"]] = neck_angle
+
+        # Spine
+        spine_angle = spi_an(pelvis_axis, thorax_axis)
+        angle_results[angle_idx["Spine"]] = spine_angle
+
+        # Shoulder
+        right_shoulder_angle, left_shoulder_angle = sho_an(thorax_axis, elbow_axis)
+        angle_results[angle_idx["R Shoulder"]] = right_shoulder_angle
+        angle_results[angle_idx["L Shoulder"]] = left_shoulder_angle
+
+        # Elbow
+        right_elbow_angle, left_elbow_angle = elb_an(elbow_axis, wrist_axis)
+        angle_results[angle_idx["R Elbow"]] = right_elbow_angle
+        angle_results[angle_idx["L Elbow"]] = left_elbow_angle
+
+        # Wrist
+        right_wrist_angle, left_wrist_angle = wri_an(wrist_axis, hand_axis)
+        angle_results[angle_idx["R Wrist"]] = right_wrist_angle
+        angle_results[angle_idx["L Wrist"]] = left_wrist_angle
+
+        # Center of Mass calculations
+        com_results = np.array(CGM.get_kinetics(joint_centers, jc_idx, seg_scale, measurements["Bodymass"]))
+
+        return axis_results, angle_results, com_results, joint_centers
+
+    # Utility functions
```

Comment:
if this comment is referring to some 'section' of functions to come, maybe putting some -----------------  before/ after to make a clear section marker would help